src/packages/documents/documents/components/input-document-source/input-document-source.element.ts

```diff
@@ -0,0 +1,108 @@
+import { UmbDocumentPickerContext } from '../input-document/input-document.context.js';
+import { css, html, customElement, property, state, ifDefined, repeat } from '@umbraco-cms/backoffice/external/lit';
+import { FormControlMixin, UUIButtonElement } from '@umbraco-cms/backoffice/external/uui';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import type { DocumentItemResponseModel } from '@umbraco-cms/backoffice/backend-api';
+
+@customElement('umb-input-document-source')
```

Comment:
'doucment-source' is an okay name, but its just not something we use anywhere for this matter. Could this be named 'document-root'?, eventually 'document-picker-root'.
All in favour to align with 'Dynamic Root'? What do you think about that?
:-)

Reply:
@nielslyngsoe I agree, 'document-picker-root' is a better name. I'll update accordingly.